jekyll/_cci2/env-vars.md

```diff
@@ -85,13 +85,14 @@ jobs:
 CircleCI does not support interpolation of environment variables.
 All defined values are treated literally.
```

Comment:
This may be confusing - inside of shell commands env vars do resolve. We don't support interpolation in the configuration variables (eg: working_directory, images, etc.)

Reply:
Great point. I'll get that clarified.